src/core/main.c

```diff
@@ -1390,8 +1390,11 @@ static int bump_unix_max_dgram_qlen(void) {
          * $NOTIFY_SOCKET one. */
 
         r = read_one_line_file("/proc/sys/net/unix/max_dgram_qlen", &qlen);
-        if (r < 0)
+        if (r < 0) {
+                if (errno == ENOENT)
+                        return 0;
                 return log_warning_errno(r, "Failed to read AF_UNIX datagram queue length, ignoring: %m");
```

Comment:
hmm, let's always log about this please, but downgrade the log level to debug if it's ENOENT.

Also, we generally follow the logic that our functions return negative error numbers, i.e. you need to check `r` against `-ENOENT`, and not `errno` to `EAGAIN`…

something like this hence:
```c
if (r < 0) {
        return log_full_errno(r == -ENOENT ? LOG_DEBUG : LOG_WARNING, r, …
```

Reply:
thanks, I've amended it and tested locally